### DISCUSSION TITLE
R#022 互联网对立情绪的建议

### DISCUSSION THREAD

**Discussion by giscus** [lc4t/lc4t.github.io#28] [Announcements]
# R#022 互联网对立情绪的建议

2024.10.29 ~ 2024.11.04

https://blog.sakanano.moe/journals/random_022/

**Comment by wuTong0369** [comment]
博主买股票吗,有什么好的趁手工具推荐的吗

**Reply by lc4t** [reply]
要说买股票的话一般A档券商都是可以的：http://www.csrc.gov.cn/tianjin/c105376/cbecbcac9a84445e6bf4d332ad4e50424/content.shtml；
如果是要分析股票的话还是要先学基础只是，不然连treadingview的数据都看不懂...treadingview应该算是最好用的工具了
